packages/pn-personafisica-login/package.json

```diff
@@ -59,27 +55,30 @@
   "devDependencies": {
     "@babel/core": "^7.23.2",
     "@babel/preset-env": "^7.23.2",
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
+    "@babel/preset-react": "^7.23.2",
+    "@babel/preset-typescript": "^7.23.2",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.5.1",
     "@trivago/prettier-plugin-sort-imports": "^4.2.0",
-    "@types/history": "^4.7.9",
-    "@types/jest": "^26.0.15",
-    "@types/react-router": "^5.1.16",
     "@typescript-eslint/eslint-plugin": "^6.7.3",
     "@typescript-eslint/parser": "^6.7.3",
-    "babel-loader": "^8.0.2",
+    "@vitejs/plugin-basic-ssl": "^1.0.1",
+    "@vitejs/plugin-react": "^4.0.3",
+    "@vitest/coverage-v8": "0.34.6",
     "eslint": "7.11.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-functional": "^3.7.2",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-sonarjs": "^0.10.0",
-    "jest-sonar-reporter": "^2.0.0",
+    "identity-obj-proxy": "^3.0.0",
     "prettier": "^2.4.1",
-    "react-scripts": "^5.0.1",
     "sonarqube-scanner": "^3.0.0",
-    "typescript": "^5.2.2"
+    "vite": "^4.4.5",
+    "vitest": "0.34.6",
+    "vitest-sonar-reporter": "0.5.0",
+    "whatwg-fetch": "^3.6.2"
```

Comment:
Some tests of pn-personafisica-login involve in an indirect way the component `FooterPreLogin`, included in `mui-italia`. This component uses `fetch`, is the only component in `mui-italia` to do so.
When running the tests in the Vite-based project, such tests fail because the definition of `fetch` yields an error.
The use of `whatwg-fetch`, which is imported in `setupTests.ts`, avoid such errors.

In the CRA-based project (that is still accessible in `develop`), this package is indeed included in `yarn.lock` through the following path: `react-scripts` --> `react-app-polyfill` --> `whatwg-fetch`.